Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/BoundingBoxes/BoundingBoxEnums/BoundingBoxGizmoHandleAxisToAffectEnum.cs

```diff
@@ -3,9 +3,12 @@
 
 namespace Microsoft.MixedReality.Toolkit.SDK.UX
 {
+    /// <summary>
+    /// Which axis does a BoundingBoxRigGizmoHandle rotate around?
+    /// </summary>
     public enum BoundingBoxGizmoHandleAxisToAffect
```

Comment:
This seems really generic, can we possibly move this into core?

Reply:
sure! where in Core would be the best place?

Reply:
I moved this enum file to Core->Definitions->Utilities and renamed CardinalAxisType